js/build.js

```diff
@@ -263,6 +280,8 @@ Fliplet.Widget.instance('chat', function (data) {
 
     getContacts(false).then(function () {
       return getConversations();
+    }).catch(function(error) {
```

Comment:
@squallstar This is what fixes that message. I was doing the catch in the wrong place. (Maybe it's still not entirely correct)